fuseoverlayfs.go

```diff
@@ -182,9 +181,12 @@ func (o *snapshotter) Mounts(ctx context.Context, key string) ([]mount.Mount, er
 	if err != nil {
 		return nil, err
 	}
+	defer t.Rollback()
```

Comment:
Moved just for readability / matches the standard practice of other functions in this snapshotter.

Reply:
This should be another PR